packages/clerk-js/src/ui/styledSystem/animations.ts

```diff
@@ -32,6 +32,37 @@ const fadeIn = keyframes`
   100% { opacity: 1; }
 `;
 
+const inAnimation = keyframes`
+  0% {
+    opacity: 0;
+    transform: translateY(-5px);
+    max-height: 0;
+  }
+  100% {
+    opacity: 1;
+    transform: translateY(0px);
+    max-height: 6rem;
```

Comment:
Since this is an animation we might want to use for taller elements, should we do something like:
`max-height: var(--clerk-collapse-max-height, 6rem)` ?

Reply:
This will probably work, but imagine this scenario:

- We define `--clerk-collapse-max-height` to be 10rem
-  There are 2 places in the screen where this animation will run.
-  One item has a shorter height
- 2nd item has 10rem height.
We managed to run the animation as intended for the 2nd item, but now the animation for the 1st item will run faster